src/components/DashboardHome/SidebarBlock.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+const SidebarBlock = props => (
+  <section className={classNames(props.sectionClassName)}>
```

Comment:
You can rename `sectionClassName` to just `className`. And since it's just a string passed in, we don't really need to use `classNames` here: `<section className={props.className}>`